astropy/nddata/utils.py

```diff
@@ -69,6 +75,9 @@ def overlap_slices(large_array_shape, small_array_shape, position, mode="partial
         otherwise an `~astropy.nddata.utils.PartialOverlapError` is
         raised.  In all modes, non-overlapping arrays will raise a
         `~astropy.nddata.utils.NoOverlapError`.
+    limit_rounding_method : {'ceil', 'round' or 'floor'}
+        Allows the user to choose the rounding method of their own choice when trimming the pixel indices.
```

Comment:
```suggestion
        The rounding method when calculating the minimum and maximum pixel indices.
```
Suggestion to match the style of the rest of the parameter descriptions.